relay-dynamic-config/src/defaults.rs

```diff
@@ -263,6 +280,30 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                 Tag::with_key("span.status_code")
                     .from_field("span.sentry_tags.status_code")
                     .when(is_http.clone()),
+                // Cache module
+                Tag::with_key("cache.hit")
+                    .from_field("span.sentry_tags.cache_hit")
+                    .when(is_cache.clone()),
+            ],
+        },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/cache.item_size@byte".into(),
```

Comment:
@Dav1dde @phacops what's the status on [strings.py](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/strings.py)? Do we still want to add metric names there?

Reply:
I think that has been "deprecated" and it isn't supposed to be changed anymore, there is a Python test preventing modifications using a hashsum of the file.

Although I think it's a bit unfortunate because it makes querying for specific metrics in clickhouse directly impossible :(

Reply:
IMO we should still allow MRIs to be added to that file. The chance of those appearing in existing customer data is extremely low, and it makes running cardinality queries much easier, like you said.